src/app/Dashboard/AutomatedAnalysis/AutomatedAnalysisCard.tsx

```diff
@@ -0,0 +1,759 @@
+  ]);
+
+  const startProfilingRecording = React.useCallback(() => {
+    const config = context.settings.automatedAnalysisRecordingConfig();
+    const attributes = automatedAnalysisConfigToRecordingAttributes(config);
+    addSubscription(
+      context.api.createRecording(attributes).subscribe((resp) => {
+        if (resp && (resp.ok || resp.status === 400)) {
+          // in-case the recording already exists
+          generateReport();
+        } else {
+          handleStateErrors(RECORDING_FAILURE_MESSAGE);
+        }
+      })
+    );
+  }, [
+    addSubscription,
+    context.api,
+    context.settings,
+    context.settings.automatedAnalysisRecordingConfig,
+    generateReport,
+    handleStateErrors,
+  ]);
+
+  const getMessageAndRetry = React.useCallback(
+    (errorMessage: string | undefined): [string | undefined, undefined | (() => void)] => {
+      if (errorMessage) {
+        if (errorMessage === NO_RECORDINGS_MESSAGE) {
+          return [undefined, undefined];
+        } else if (isAuthFail(errorMessage)) {
+          return ['Retry', generateReport];
+        } else if (errorMessage === RECORDING_FAILURE_MESSAGE) {
+          return ['Retry starting recording', startProfilingRecording];
+        } else if (errorMessage === FAILED_REPORT_MESSAGE) {
+          return ['Retry loading report', generateReport];
+        } else {
+          return ['Retry', generateReport];
+        }
+      }
+      return [undefined, undefined];
+    },
+    [startProfilingRecording, generateReport]
+  );
+
+  React.useEffect(() => {
+    context.target.target().subscribe((target) => {
+      setTargetConnectURL(target.connectUrl);
+      dispatch(automatedAnalysisAddTargetIntent(target.connectUrl));
+      generateReport();
+    });
+  }, [context.target, generateReport, setTargetConnectURL, dispatch]);
+
+  React.useEffect(() => {
+    if (reportTime == 0 || !(usingArchivedReport || usingCachedReport)) {
+      return;
+    }
+    const analysisTimer = calculateAnalysisTimer(reportTime);
+    setReportStalenessTimer(analysisTimer.quantity);
+    setReportStalenessTimerUnits(analysisTimer.unit);
+    const timer = setInterval(() => {
+      setReportStalenessTimer((reportStalenessTimer) => reportStalenessTimer + 1);
+    }, analysisTimer.interval);
+    return () => clearInterval(timer);
+  }, [
+    setReportStalenessTimer,
+    setReportStalenessTimerUnits,
+    reportTime,
+    reportStalenessTimer,
+    usingArchivedReport,
+    usingCachedReport,
+  ]);
+
+  React.useEffect(() => {
+    setFilteredCategorizedEvaluation(
+      filterAutomatedAnalysis(
+        categorizedEvaluation,
+        targetAutomatedAnalysisFilters,
+        targetAutomatedAnalysisGlobalFilters,
+        showNAScores
+      )
+    );
+  }, [
+    categorizedEvaluation,
+    targetAutomatedAnalysisFilters,
+    targetAutomatedAnalysisGlobalFilters,
+    showNAScores,
+    filterAutomatedAnalysis,
+    setFilteredCategorizedEvaluation,
+  ]);
+
```

Comment:
We should a listener for authRetry and generateReport again like this:

```typescript
  React.useEffect(() => {
    addSubscription(context.target.authRetry().subscribe(generateReport));
  }, [context.target, addSubscription])
```

Reply:
This way, when the user enters the credentials correctly, the analysis should pop up (if any).